pimd/pim_cmd_common.c

```diff
@@ -2316,6 +2316,7 @@ void pim_show_interfaces(struct pim_instance *pim, struct vty *vty, bool mlag,
 				       address, neighbors, pimdr, firsthpr,
 				       pimifchnl);
 		}
+		json_object_free(json);
```

Comment:
Isn't this handled by `vty_json()` in parent functions? Like `pim_show_interface_cmd_helper` or `pim_show_interface_vrf_all_cmd_helper`. For instance:

```c
	else
		pim_show_interfaces(v->info, vty, mlag, json_parent);

	if (uj)
		vty_json(vty, json_parent);
```
In this example, `vty_json()`, `json_parent` will be freed in `pim_show_interfaces()`?

Reply:
Yes it is handled that way for json but here it is being used for CLI, therefore it should be freed in case of CLI. This is to free the memory allocated at Line 2234.